### PR TITLE
HW#1 Max 10/19/2019

### PR DESCRIPTION
**Pull request by yegorovmax**
HW#1 
 1. Created a function "isPrime" that takes an integer and returns a boolean which indicates if the number is prime
 2. Created a function "printAllPrimeNumbersBeforeGivenNumber"  that takes an integer and prints all prime numbers before this integer

### CODE REVIEW

File: MyPlayground.playground/Contents.swift [hunk 1]
```diff
@@ -3,3 +3,45 @@ import UIKit
 /*
  1. Create a function that takes an integer and returns a boolean which indicates if the number is prime
  2. (optional for extra bonus) Create a function that takes an integer and prints all prime numbers before this integer */
+
+func isPrime(number: Int) -> Bool {
+    if number > 1 && number < 4 {
+        return true
+    }
+    if number > 1 && number / 1 == number {
```
Comment (booorisss) [inline MyPlayground.playground/Contents.swift:11]: `number / 1 == number` is always true 😄

Comment (booorisss) [reply MyPlayground.playground/Contents.swift:11]: and `number > 1` is already checked at the line 8

**Review by booorisss — COMMENTED** [review]
It does work correctly 👍 However, it is too verbose. Half of those checks are not needed 😄

File: MyPlayground.playground/Contents.swift (hunk 1 above)
Comment (yegorovmax) [reply MyPlayground.playground/Contents.swift:11]: I can remove this check "number / 1 == number"
second if else i need to check if number is negative

Comment (yegorovmax) [reply MyPlayground.playground/Contents.swift:11]: i still have to have one more if else to check if number > 1 and return FALSE if its not true

**Pull request closed without merging**